ui/media/main.js

```diff
@@ -262,6 +263,31 @@ async function healthCheck() {
         setStatus('server', 'offline', 'error')
     }
 }
+function resizeInpaintingEditor() {
+    let widthValue = parseInt(widthField.value)
+    let heightValue = parseInt(heightField.value)
+    if (widthValue === heightValue) {
+        widthValue = INPAINTING_EDITOR_SIZE
+        heightValue = INPAINTING_EDITOR_SIZE
+    } else if (widthValue > heightValue) {
+        heightValue = (heightValue / widthValue) * INPAINTING_EDITOR_SIZE
+        widthValue = INPAINTING_EDITOR_SIZE
+    } else {
+        widthValue = (widthValue / heightValue) * INPAINTING_EDITOR_SIZE
+        heightValue = INPAINTING_EDITOR_SIZE
+    }
+    inpaintingEditorContainer.setAttribute("style", `width:${widthValue}px;height:${heightValue}px`)
```

Comment:
Why is this line set, if we're changing the style.width and height in the next two lines?

Plus, this line will overwrite any custom styling (which we shouldn't do, and don't do). But is this line really necessary?

Reply:
This is to resize the div `inpaintingEditorContainer`
That's that standard way to resize a div if you need to support both Chrome and Firefox.
https://stackoverflow.com/a/10118190